packages/style/scss/helpers.scss

```diff
@@ -31,5 +31,5 @@ $browser-context: 13; // Default
     border: $border-width solid var(--cerulean-blue-50);
     border-radius: $border-radius;
     outline: none;
-    box-shadow: 0 0 0 3px var(--cerulean-blue-20);
+    box-shadow: 0 0 0 4px var(--cerulean-blue-20);
```

Comment:
cerulean is a deprecated blue right? :thinking:

Reply:
It is not
<img width="1113" alt="image" src="https://user-images.githubusercontent.com/35579930/157332428-9ba937ac-1e76-4270-9c7d-c77c577f0184.png">